gfwanalysis/services/analysis/population_service.py

```diff
@@ -0,0 +1,37 @@
+"""POPULATION SERVICE"""
+
+import logging
+
+import ee
+from gfwanalysis.errors import PopulationError
+from gfwanalysis.config import SETTINGS
+from gfwanalysis.utils.geo import get_region, squaremeters_to_ha, admin_0_simplify
+
+
+class PopulationService(object):
+
+    @staticmethod
+    def analyze(geojson):
+        """For a given geometry and population density data
+        return a dictionary of total t/ha.
+        """
+        try:
+            d = {}
+            # The estimated number of persons per square kilometer
+            population_asset = SETTINGS.get('gee').get('assets').get('population')
+            region = get_region(geojson)
+            reduce_args = {'reducer': ee.Reducer.sum().unweighted(),
+                           'geometry': region,
+                           'bestEffort': True,
+                           'scale': 30}
+            # Convert m2 to km2
+            scale_factor = ee.Number(1e6)               
```

Comment:
Really pains the physicist in me to say this, but can we scale it to hectares (1e4) since on the platform the analyses are always relative to that unit?

e.g. if someone analyses the pop layer against some other dataset there will be a clash in the area units.

Reply:
going to review this PR and make that change while I am doing it.